lib/Common/ConfigFlagsList.h

```diff
@@ -384,6 +384,7 @@ PHASE(All)
 #else
 #define DEFAULT_CONFIG_BgJitDelay           (30)
 #endif
+#endif
```

Comment:
Can we add some comments matching the #endif to the #ifdef ?

Reply:
Yeah -- it was weird, I just notice this, but a ton of the DEFAULT_CONFIGS were gated on BgJitDelay, which seemed wrong.